swarm/swarm.go

```diff
@@ -199,20 +199,24 @@ func (self *Swarm) Start(net p2p.Server) error {
 
 	log.Warn(fmt.Sprintf("Starting Swarm service"))
 	
-	glog.V(logger.Warn).Infof("Starting Swarm service")
 	self.hive.Start(
-		connectPeer,
+		net,
 		func () <-chan time.Time{
 			return time.NewTicker(time.Second).C
 		},
 	)
 
-	log.Info(fmt.Sprintf("Swarm network started on bzz address: %v", self.hive.Addr()))
+	log.Info(fmt.Sprintf("Swarm network started on bzz address: %v", self.hive.GetAddr()))
 
 	if self.pssEnabled {
 		pssparams := network.NewPssParams()
 		self.pss = network.NewPss(self.hive.Overlay, pssparams)
-		glog.V(logger.Info).Infof("Pss started: %v", self.pss)
+		
```

Comment:
I know you mentioned you put this in swarm, but this must be temporary. ok I see you left exactly this comment. All cool